applications/StructuralMechanicsApplication/tests/test_patch_test_small_strain.py

```diff
@@ -135,11 +135,10 @@ def _check_results(self,mp,A,b):
 
             d = node.GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT)
             for i in range(3):
-                if abs(u[i]) > 0.0:
-                    error = (d[i] - u[i])/u[i]
-                    if error > 1.0e-6:
-                       print("NODE ", node.Id,": Component ", coor_list[i],":\t",u[i],"\t",d[i], "\tError: ", error)
-                    self.assertLess(error, 1.0e-6)
+                error = abs((d[i] - u[i])/u[i])
+                if error > self.tolerance:
+                    print("NODE ", node.Id,": Component ", coor_list[i],":\t",u[i],"\t",d[i], "\tError: ", error)
+                self.assertLess(error, self.tolerance)
```

Comment:
```suggestion
                self.assertLess(error, self.tolerance, msg=f"NODE {node.Id}: Component {coor_list[i]}: {u[i]} {d[i]} Error: {error}")
```

Assertions have an ```msg``` keyword argument you can use here, and Python36 f-strings make it prettier too!